editor/js/Sidebar.Object.js

```diff
@@ -259,6 +259,26 @@ Sidebar.Object = function ( editor ) {
 
 	container.add( objectVisibleRow );
 
+	// frustumCulled
+
+	var objectFrustumCulledRow = new UI.Row();
+	var objectFrustumCulled = new UI.Checkbox().onChange( update );
+
+	objectFrustumCulledRow.add( new UI.Text( 'Frustum Culled' ).setWidth( '90px' ) );
+	objectFrustumCulledRow.add( objectFrustumCulled );
+
+	container.add( objectFrustumCulledRow );
+
+	// renderOrder
+
+	var objectRenderOrderRow = new UI.Row();
+	var objectRenderOrder = new UI.Integer().setWidth( '50px' ).setStep( 1 ).setRange( 0, Infinity ).onChange( update );
```

Comment:
Hmm... I guess the range should go from `-Infinity` to `Infinity`?
So I guess we don't need to set the range?

Reply:
Right. `.setStep()` is also not necessary 😅